TLM/TLM/Manager/Impl/SpeedLimitManager.cs

```diff
@@ -696,8 +734,10 @@ public class SpeedLimitManager
                 }
             }
 
+            Log.Info("SpeedLimitManager.OnBeforeLoadData: Scan complete");
+
             mainNetInfos.Sort(
-                delegate(NetInfo a, NetInfo b) {
+                delegate (NetInfo a, NetInfo b) {
```

Comment:
Does your IDE suggest making a lambda here? Essentially delete the `delegate` and add a `=>` before the `{`

Reply:
There's a gazillion recommendations in that file from the IDE, let me check.

Yes, it does. I'll send another commit lambda-ified :) I'll clear up a few of the other things it's suggesting too. :)

As for the `nextiter:` I have no clue how to clean that up. Last time I used 'goto' labels was in... wow, uhm... Something on Gem Desktop on an Amstrad 1640 (640 kB of RAM and 360 kB floppy disk!!) when I was about (guessing) 6 years old...

Reply:
There's a load of references to obsolete `Flags` from `TLM/TLM/State/Flags.cs` - not sure what to do about that...

Reply:
Just avoid using flags in the new code, and they will eventually be phased out. I kind of understand what Victor wanted to do with multiple new "Manager" classes, when it is done, the Flags will be an empty class.